packages/access/src/tests/test_ownable_twostep.cairo

```diff
@@ -244,6 +244,22 @@ fn test_renounce_ownership() {
     assert!(state.owner().is_zero());
 }
 
+#[test]
+fn test_renounce_ownership_resets_pending_owner() {
```

Comment:
This looks good. Should we also add a test that asserts that under this circunstance if you try to transfer ownership it panics with `NOT_PENDING_OWNER`?